src/base/ReactorStructs.sol

```diff
@@ -13,10 +13,18 @@ struct InputTokenWithRecipient {
     address recipient;
 }
 
+/// @dev tokens that need to be received by the recipient in order to satisfy an order
+struct OutputToken {
```

Comment:
how come we don't call this `OutputTokenWithRecipient` or how come its not just `OutputToken` and `InputToken`?

Reply:
So I left it as OutputToken since it's essentially the same struct as the one in UniswapX, as compared to InputTokenWithRecipient, which is different than the `InputToken` in UniswapX. Open to changing the naming around